com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -440,8 +530,6 @@ void DrawRendererListLayout(ReorderableList list, SerializedProperty prop)
 
             list.onCanRemoveCallback = li => { return li.count > 1; };
 
-            list.onCanAddCallback = li => { return li.count < UniversalRenderPipeline.maxScriptableRenderers; };
```

Comment:
Yes. The renderer limit was 8 and we've hit a wall in our URP test project because of that. I talked to Andre a while ago and he couldn't remember why there was a limit in the first place.